portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,57 @@
+package com.google.sps.data;
+
+import java.util.Date;
+import com.google.appengine.api.datastore.Entity;
+
+
+/** Class containing a user's comment. */
```

Comment:
nit: "contains" a user's comment is technically not accurate. It is an abstraction of a user comment. It's not a container class or some sort of data structure